data/campaigns/Sceptre_of_Fire/utils/rune-equip.cfg

```diff
@@ -0,0 +1,497 @@
+                                variable=unit
+                                find_vacant=no
+                            [/unstore_unit]
+                            [remove_item]      
+                                halo="scenery/summoning-circle[4,5,6].png:[250,250,250]"
+                                x,y=$x1|,$y1|
+                            [/remove_item]
+                        [/command]
+                    [/option]
+                    [option]
+                        {SOF_RUNIC_OPTION swiftness (_ "Swiftness") (_ "Adds 2 MP") ("misc/rune_icon.png~BLIT(scenery/summoning-circle4.png~CROP(6,6,60,60),0,0)") 13 (
+                                        [effect]
+                                            apply_to=movement
+                                            increase=2
+                                        [/effect]
+                        )}
+                    [/option]
+                    [option]
+                        {SOF_RUNIC_OPTION accuracy (_ "Accuracy") (_ "Increases ranged weapon accuracy 20%") ("misc/rune_icon.png~BLIT(scenery/summoning-circle5.png~CROP(6,6,60,60),0,0)") 16 (
+                                        [effect]
+                                            apply_to=attack
+                                            range=ranged
+                                            increase_accuracy=20
+                                        [/effect]
+                        )}
+                    [/option]
+                    [option]
+                        {SOF_RUNIC_OPTION "force" (_ "Force") (_ "Increases melee weapon damage by 4") ("misc/rune_icon.png~BLIT(scenery/summoning-circle6.png~CROP(6,6,60,60),0,0)") 14 (
+                                        [effect]
+                                            apply_to=attack
+                                            range=melee
+                                            increase_damage=4
+                                        [/effect]
+                        )}
+                    [/option]
+                [/message]
+            [/event]
+[/event]
+
+#enddef
+
+#define SOF_RUNIC_CHEST_GENERIC OPTION_WML X Y
+
+[event]
+    name=prestart
+    [item]
+         x,y={X},{Y}
+         image=items/chest-plain-closed.png
+    [/item]
+    [set_variable]
+        name=chest_{X}_{Y}
+        value=yes
+    [/set_variable]
+[/event]
+
+[event]
+    name=moveto
+    first_time_only=no
+    [filter]
+        side=1
+        race=dwarf
+        x,y={X},{Y}
+    [/filter]
+    [filter_condition]
+        [variable]
+            name=chest_{X}_{Y}
+            equals=yes
+        [/variable]
+    [/filter_condition]
+        {CLEAR_VARIABLE chest_{X}_{Y}}
+        [message]
+            speaker=unit
+            message= _ "This is part of our stash of field runes, we can use them to get a small temporary boost.  We don't have very many though, and need to contribute funds for replacements.  should I take this one?" 
```

Comment:
Missing capitalization for "should". Also, double spaces here and in all other messages.